resources/dicts/events/misc/general.json

```diff
@@ -1,3722 +1,3943 @@
-    "sub_type": ["accessory"],
-    "weight": 20,
-    "event_text":  "r_c found a little patch of acc_plural and decided to gift them to m_c",
-    "new_accessory": ["POPPY", "ORANGE POPPY", "CYAN POPPY", "WHITE POPPY", "PINK POPPY", "BULB WHITE", "BULB YELLOW", "BULB ORANGE", "BULB PINK", "BULB BLUE", "LAVENDER", "HEATHER", "GORSE", "LILY OF THE VALLEY", "BLUEBELLS", "FORGET ME NOTS"],
-    "m_c": {
-      "age": ["adolescent", "young adult", "adult", "senior adult", "senior"]
-    },
-    "r_c": {
-      "age": ["adolescent", "young adult", "adult", "senior adult", "senior"]
-    }
-  },
-  {
-    "event_id": "gen_misc_mate_accessory_bud",
-    "location": ["any"],
-    "season": ["newleaf","greenleaf"],
-    "subtype": ["accessory", "romance"],
-    "weight": 10,
-    "event_text": "Flowers can be so pretty, even before they bloom, just like m_c. r_c blushes as they see m_c wear the acc_singular {PRONOUN/r_c/subject} got {PRONOUN/m_c/object}",
-    "new_accessory": ["BULB WHITE", "BULB YELLOW", "BULB ORANGE", "BULB PINK", "BULB BLUE"],
-    "m_c": {
-      "age": ["young adult", "adult", "senior adult"]
-    },
-    "r_c": {
-      "age": ["young adult", "adult", "senior adult"]
-    },
-    "relationships": [
-        {
-          "cats_to": ["m_c"],
-            "cats_from": ["r_c"],
-            "mutual": true,
-            "values": ["romantic", "comfort", "trust"],
-          "amount": 10
-        }
-    ]
-  },
-  {
-    "event_id": "gen_misc_any_medicine_omen_bloodthristy",
-    "location": ["any"],
-    "season": ["any"],
-    "tags": ["skill_trait_required"],
-    "weight": 5,
-    "event_text": "m_c thinks starclan is trying to warn {PRONOUN/m_c/object} about r_c.",
-    "m_c": {
-      "age": ["adolescent", "young adult", "adult", "senior adult", "senior"],
-      "status": ["medicine cat"],
-      "skill": ["OMEN,0"]
-    },
-    "r_c": {
-      "trait": ["bloodthristy"]
-    },
-    "relationships": [
-        {
-          "cats_to": ["m_c"],
-            "cats_from": ["r_c"],
-            "mutual": true,
-            "values": ["dislike"],
-          "amount": 20
-        }
-    ]
-  },
-  {
-    "event_id": "gen_misc_any_medicine_accessory_bug",
-    "location": ["any"],
-    "season": ["leaf-bare"],
-    "sub_type": ["accessory"],
-    "weight": 10,
-    "event_text": "m_c finds a stray acc_singular in its last moments of life leading them to a new herb gathering spot before {PRONOUN/m_c/subject} {VERB/m_c/have/has} never found before. It gives a final flutter before it gently decends to the ground. Is it a sign from star clan, or a coincidence? m_c doesnt know but {PRONOUN/m_c/subject} {VERB/m_c/decide/decides} to keep it with {PRONOUN/m_c/object} in its honor.",
-    "new_accessory": ["BLACK CICADA", "CICADA WINGS", "ROSY MOTH WINGS", "MORPHO BUTTERFLY", "MONARCH BUTTERFLY"],
-    "m_c": {
-      "age": ["adolescent", "young adult", "adult", "senior adult", "senior"],
-      "status": ["medicine cat"]
```

Comment:
i know it's probably just formatter weirdness, but just confirm for me that there hasn't been a bunch of acc events deleted somehow